frontend/src/containers/AddChart.tsx

```diff
@@ -103,6 +103,7 @@ function AddChart() {
   const [currencyTypes, setCurrencyTypes] = useState<
     Map<string, CurrencyDataType>
   >(new Map<string, CurrencyDataType>());
+  const [oldStep, setoldStep] = useState<number>(-1);
```

Comment:
Should we name the setter setOldStep since TypsScript uses camelCase? This also happens in other files below.

Reply:
yes, good catch!